UefiPayloadPkg/Library/PciSegmentInfoLibAcpiBoardInfo/PciSegmentInfoLibAcpiBoardInfo.c

```diff
@@ -13,48 +15,175 @@
 #include <Library/HobLib.h>
 #include <Library/PciSegmentInfoLib.h>
 #include <Library/DebugLib.h>
+#include <UniversalPayload/PciRootBridges.h>
+#include <Library/PciLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/PcdLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Guid/PciSegmentInfoGuid.h>
+
+static PCI_SEGMENT_INFO  *PciSegments = 0;

```

Comment:
just picky comments, please use mPciSegments for modular global variable to align with coding style.

Reply:
done